charts/shoot-core/components/charts/network-policies/templates/allow-to-dns.yaml

```diff
@@ -20,6 +20,10 @@ spec:
     - podSelector:
         matchExpressions:
         - {key: k8s-app, operator: In, values: [kube-dns,coredns]}
+    {{- if .Values.nodeLocalDNS.enabled }}
+    - ipBlock:
+        cidr: {{ .Values.nodeLocalDNS.kubeDNSClusterIP }}/32
+    {{- end }}
```

Comment:
IIUC, this is only needed if Kube-Proxy runs in `IPTables` mode. Should we consider this case here, too?

Reply:
Btw. is `169.254.20.10` allowed by default?

Reply:
> Btw. is 169.254.20.10 allowed by default?

Yes